scripts/licenseChecker.js

```diff
@@ -0,0 +1,51 @@
+const checker = require('license-checker');
+const path = require('path');
+
+// The licenses prohibited from use at Amazon - https://policy.a2z.com/docs/82477/publication
```

Comment:
Should we include an internal link here?

Reply:
Good point, will remove it

Reply:
Maybe also do a force-push to change the history?